aries-site/src/examples/templates/forms/SettingsExample.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Main,
+  MaskedInput,
+  RangeInput,
+  Text,
+  TextInput,
+} from 'grommet';
+
+import { FormContainer } from '.';
+
+export const SettingsExample = () => {
+  const [formValues, setFormValues] = React.useState({
+    doNotDisturbHours: '9:00pm - 5:00am',
+    notifications: true,
+    notificationsVolume: 0.7,
+  });
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Heading level={3} margin="none">
+            Settings
+          </Heading>
+          <Text>for HPE Service</Text>
+        </Header>
+        <Main
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
+            value={formValues}
+            onChange={setFormValues}
+            onSubmit={({ value, touched }) => onSubmit({ value, touched })}
+          >
+            <Box margin={{ bottom: 'small' }}>
+              <Text margin={{ left: 'small' }}>Key feature</Text>
+              <FormField
+                name="notifications"
+                label="Notifications"
+                component={CheckBox}
```

Comment:
Noticed the "on" toggle shouldn't be filed in black from what I see in the sticker sheet. Should look the same as "off" toggle

Expected look:
<img width="128" alt="Screen Shot 2020-03-05 at 9 03 04 AM" src="https://user-images.githubusercontent.com/12522275/76005925-91bc7400-5ec0-11ea-83f6-479a4ebe1f1f.png">

Reply:
@taysea I could use a second set of eyes on applying that styling. It's a bit of a "rabbit hole" and ended up eating a lot of time.  Perhaps we can get on a call?

Reply:
Yeah, let's do it !